{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -1,36 +1,53 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python:
+{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
+  - "{{ env }}"
+{% endfor %}
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
+{% endif %}
   matrix:
-    - TOXENV=check
-{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
-    - TOXENV={{ env }}-cover
+    - COVERAGE=cover
       {%- if cookiecutter.coveralls|lower == 'yes' %}
         {%- if cookiecutter.c_extension_support|lower == "yes" %},extension-coveralls{% endif %},coveralls
       {%- endif %}
       {%- if cookiecutter.codecov|lower == 'yes' %},codecov{% endif %}
-    - TOXENV={{ env }}-nocov
-{% endfor %}
+    - COVERAGE=nocov
+matrix:
+  include:
+    - python: "3.4"
+      env: TOXENV=check
+
 before_install:
-  - python --version
```

Comment:
The versions are intentionally shown as they are not equivalent with what's in the build log header.

They have helped me debug issues with caches and other pip install issues (pip is basically broken if you have eggs around). I have been burned too many times :)